app/src/main/cpp/skyline/gpu/interconnect/maxwell_3d/pipeline_manager.cpp

```diff
@@ -878,24 +878,26 @@ namespace skyline::gpu::interconnect::maxwell3d {
         });
     }
 
-    PipelineManager::PipelineManager(GPU &gpu) {
-        std::ifstream stream{gpu.graphicsPipelineCacheManager->OpenReadStream()};
-        i64 lastKnownGoodOffset{stream.tellg()};
-        try {
-            auto startTime{util::GetTimeNs()};
-            PipelineStateBundle bundle;
-
-            while (bundle.Deserialise(stream)) {
-                lastKnownGoodOffset = stream.tellg();
-                auto accessor{FilePipelineStateAccessor{bundle}};
-                map.emplace(bundle.GetKey<PackedPipelineState>(), std::make_unique<Pipeline>(gpu, accessor, bundle.GetKey<PackedPipelineState>()));
-            }
+    PipelineManager::PipelineManager(GPU &gpu, bool enabled) {
+        if (enabled){
```

Comment:
You should probably use a early exit here to avoid one more useless indentation level, do like the following:
```cpp
if (!enabled)
  return;
```

Reply:
Rename this to 'loadDiskCache'